--- dwm/config.h.orig	2016-10-24 14:00:38.680884777 +0500
+++ dwm/config.h	2016-10-24 16:18:53.829803458 +0500
@@ -17,7 +17,7 @@
 static const int topbar             = 1;        /* 0 means bottom bar */
 
 /* tagging */
-static const char *tags[] = { "1", "2", "3", "4", "5", "6", "7", "8", "9" };
+static const char *tags[] = { "1web", "2im", "3term", "4fm", "5", "6", "7", "8", "9" };
 
 static const Rule rules[] = {
 	/* xprop(1):
@@ -26,7 +26,9 @@
 	 */
 	/* class      instance    title       tags mask     isfloating   monitor */
 	{ "Gimp",     NULL,       NULL,       0,            1,           -1 },
-	{ "Firefox",  NULL,       NULL,       1 << 8,       0,           -1 },
+	{ "Firefox",  NULL,       NULL,       1 << 0,       0,           -1 },
+	{ "Chromium", NULL,       NULL,       1 << 0,       0,           -1 },
+	{ "Pidgin",	  NULL,		  NULL,	     1 << 1,       0,			  -1 },
 };
 
 /* layout(s) */
@@ -41,7 +43,12 @@
 	{ "[M]",      monocle },
 };
 
-/* key definitions */
+/* key definitions - ASUS A42F */
+#define XF86AudioMute		0x1008ff12
+#define XF86AudioLowerVolume	0x1008ff11
+#define XF86AudioRaiseVolume	0x1008ff13
+#define XF86PrintScreen		0xff61
+
 #define MODKEY Mod1Mask
 #define TAGKEYS(KEY,TAG) \
 	{ MODKEY,                       KEY,      view,           {.ui = 1 << TAG} }, \
@@ -55,7 +62,11 @@
 /* commands */
 static char dmenumon[2] = "0"; /* component of dmenucmd, manipulated in spawn() */
 static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", normbgcolor, "-nf", normfgcolor, "-sb", selbgcolor, "-sf", selfgcolor, NULL };
-static const char *termcmd[]  = { "st", NULL };
+static const char *termcmd[]  = { "xterm", NULL };
+static const char *incr_sound[] = { "amixer", "sset", "Master", "5%+", NULL };
+static const char *decr_sound[] = { "amixer", "sset", "Master", "5%-", NULL };
+static const char *toggle_sound[] = { "amixer", "-q", "sset", "Master", "toggle", NULL };
+static const char *screenshotcmd[] = { "scrot", "$HOME/Pictures/screenshots/%Y%m%d-%T-screenshot.png", NULL };
 
 static Key keys[] = {
 	/* modifier                     key        function        argument */
@@ -92,6 +103,10 @@
 	TAGKEYS(                        XK_8,                      7)
 	TAGKEYS(                        XK_9,                      8)
 	{ MODKEY|ShiftMask,             XK_q,      quit,           {0} },
+	{ 0,                            XF86AudioMute,        spawn,        {.v = toggle_sound } },
+	{ 0,                            XF86AudioRaiseVolume,         spawn,        {.v = incr_sound } },
+	{ 0,                            XF86AudioLowerVolume,        spawn, {.v = decr_sound } },
+	{ 0,                            XF86PrintScreen,	spawn, {.v = screenshotcmd } },
 };
 
 /* button definitions */
